ledger-api/grpc-definitions/com/daml/ledger/api/v1/event.proto

```diff
@@ -25,11 +25,12 @@ option csharp_namespace = "Com.Daml.Ledger.Api.V1";
 message Event {
   oneof event {
     CreatedEvent created = 1;
+    ExercisedEvent exercised = 2;
```

Comment:
I'd suggest adding commentary explaining what kind of `ExercisedEvent` values can be found here, and when they will be present.